queries.sql

```diff
@@ -15,4 +15,49 @@ select * from animals where neutered =true;
 -- Find all animals not named Gabumon.
 select * from animals where name!='Gabumon' ;
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
-select * from animals where weight_kg>=10.4 AND weight_kg<=17.3;
\ No newline at end of file
+select * from animals where weight_kg>=10.4 AND weight_kg<=17.3;
+
+-- starting a transaction and update animal table by setting the species column to unspecified and rollback.
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+-- Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+-- Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
+UPDATE animals SET species = 'pokemon' WHERE species = '';
+-- Commit the transaction.
+COMMIT;
```

Comment:
- [ ] Here also, before committing the transaction we should verify the change was made, then we commit. After committing the transaction we should also verify the change persists after the commit as seen in the requirements below
![Screenshot from 2023-06-05 18-02-52](https://github.com/KUBAHO3/vet-clinic-database/assets/46388113/6cf98682-cc0d-4d08-82b1-94f18dba124d)

Reply:
done